1_slices_modules_and_objects/exercises/acronym/src/lib.rs

```diff
@@ -10,7 +10,15 @@
 /// ```
 // Side note: this doc example is also tested when running `cargo test`
 pub fn make_acronym(name: &str) -> String {
-    unimplemented!()
+    let mut acronym: String = String::new();
+    let split = name.split(' ');
```

Comment:
Can also be written as
```suggestion
    let split = name.split_whitespaces();
```